specification/cognitiveservices/data-plane/Speech/VideoTranslation/preview/2024-05-20-preview/video-translation.json

```diff
@@ -0,0 +1,963 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete a translation": {
+            "$ref": "./examples/delete_translation.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "VideoTranslation"
+        ],
+        "description": "Creates a new translation.",
+        "operationId": "Translations_Create",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/TranslationPathResourceId"
+          },
+          {
+            "$ref": "#/parameters/HeaderOperationId"
+          },
+          {
+            "$ref": "#/parameters/QueryApiVersion"
+          },
+          {
+            "in": "body",
+            "name": "translation",
+            "description": "translation definition",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Translation"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/Translation"
+            },
+            "headers": {
+              "Operation-Id": {
+                "type": "string",
+                "description": "ID of the operation."
+              },
+              "Operation-Location": {
+                "description": "The location of the operation job. Use this URL to monitor operation status.",
+                "type": "string",
+                "format": "uri"
+              },
+              "retry-after": {
+                "description": "Indicate the minimum number of seconds the client should wait before polling (GETing) the status monitor URL again for an update.",
+                "type": "string",
+                "format": "uri"
+              }
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create a project": {
+            "$ref": "./examples/put_translation.json"
+          }
+        }
+      }
+    },
+    "/translations/{translation-id}/iterations": {
+      "get": {
+        "tags": [
+          "VideoTranslation"
+        ],
+        "description": "List iterations of the translation.",
+        "operationId": "Iterations_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
```

Comment:
Does this support QuerySkip, QueryMaxPageSize, QueryFilter just like translations_list?

Reply:
Doesn't support those when querying iterations of one translation, suppose not too many in one query for most case.